src/components/ec-table/ec-table.story.js

```diff
@@ -23,6 +23,7 @@ const columns = [
     title: 'Original amount',
     sortable: true,
     tooltip: 'This is info test',
+    align: 'right',
```

Comment:
I don't like the fact that we have two different props to set one behaviour. If `type` is icon, then use `text-center`, if `align` is right, then use `text-right`. Why not using a type? Can we give a meaningful name for the type of column you are trying to built in this PR?

Joining these two props will also eliminate a clash when you set align to right and type to icon.

Reply:
I tend to agree with Milos approach here

Reply:
Great. Now the type currency is used to discriminate it.